### DISCUSSION TITLE
Seeking mentor feedback on the "OpenVINO conditional compilation improvement" project proposal

### DISCUSSION THREAD

**Discussion by HikaruSadashi** [openvinotoolkit/openvino#23808] [Google Summer of Code]
Hi @zhaixuejun1993, Hi @riverlijunjie 

I'm a 3rd year software engineering student from Western University, Canada with some experience in ML research, some experience designing systems to scale and an interest in lower level optimization. I sent both of you an email with a draft of my proposal for the OpenVINO conditional compilation improvement which I see no discussion for and would love to get any feedback for especially because I just took note of this and its due in about 12 hours!

**Comment by riverlijunjie** [comment]
Hi @HikaruSadashi  I have received your proposal, it is amazing thing that you have interested in this topic and to contribute your talent to improve OpenVINO conditional compilation feature.
